lib/ReactViewModels/ViewState.js

```diff
@@ -7,6 +7,9 @@ var knockout = require('terriajs-cesium/Source/ThirdParty/knockout');
  * Root of a global view model. Presumably this should get nested as more stuff goes into it. Basically this belongs to
  * the root of the UI and then it can choose to pass either the whole thing or parts down as props to its children.
  */
+
+// mobileViews:'data', 'preview', 'nowViewing', 'search';
+
```

Comment:
Also doesn't ES2015 have an enum type?  Even if not, you can define a type with `data: 0`, `preview: 1`, and `nowViewing: 2` (or even use the strings instead of numbers) because it's less prone to typos that way.